amplify/backend/function/getMembersForColony/src/graphql.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+  getUser: /* GraphQL */ `
+    query GetUser($id: ID!) {
+      getUserByAddress(id: $id) {
+        items {
+          id
+          name
+          profile {
+            avatar
+            thumbnail
+            displayName
+            bio
+            location
+            website
+            email
+          }
+        }
+      }
+    }
+  `,
+  getWatchersInColony: /* GraphQL */ `
+    query GetWatchersInColony($id: ID!) {
+      getColonyByAddress(id: $id) {
+        items {
+          watchers {
+            items {
+              user {
+                id
+                name
+                createdAt
+                profile {
+                  avatar
+                  bio
+                  displayName
+                  email
+                  location
+                  thumbnail
+                  website
+                }
```

Comment:
Query for `profileId` here as well to fix the problem with profile being `null` when consumed by the front end.

Reply:
@willm30 Great thanks, its working well now.